c/Makefile

```diff
@@ -30,6 +31,9 @@ $(LIB_A_FILE): $(LIB_OBJ_FILES)
 example: example.o $(LIB_A_FILE)
 	$(CC) $(ALL_FLAGS) -o $@ example.o $(LD_FLAGS) -L. -l$(LIB_NAME)
 
+gonzo: gonzo.o $(LIB_A_FILE)
```

Comment:
I don't think you want to include this here :).

Reply:
Good catch, removed with b430c289e54354d3116a60dfdfac8d487d2b6cc7.